src/ResourceManager/SiteRecovery/Commands.SiteRecovery/Network/RemoveAzureRMSiteRecoveryNetworkMapping.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.Azure.Commands.SiteRecovery
     /// </summary>
     [Cmdlet(VerbsCommon.Remove, "AzureRmSiteRecoveryNetworkMapping")]
     [OutputType(typeof(ASRJob))]
-    public class RemoveAzureRMSiteRecoveryNetworkMapping : SiteRecoveryCmdletBase
```

Comment:
>RemoveAzureRMSiteRecoveryNetworkMapping [](start = 17, length = 39)

Any reason the remove Networking mapping does not have SupportShouldProcess parameter?